### ISSUE TITLE
should we remove this repo since it is not used?

### ISSUE THREAD

**Issue by indera** [ufbmi/linked#1]

**Comment by joshhanna** [comment]
My guess is this is a precursor to OLASS, so I bet we can remove this one.  I'll leave the final determination up to @bianjiang, since he created it.

**Comment by indera** [comment]
Yes, @bianjiang will probably know... I also asked him to rename https://github.com/ufbmi/olass-server to something like "olass-server-scala" or "olass-server-deprecated" in case we want to keep that code.
